README.md

```diff
@@ -1,16 +1,17 @@
 # Eclipse Graphical Editing Framework (GEF)
 
-This repository contains the code base of the [Eclipse Graphical Editing Framework (GEF)](http://www.eclipse.org/gef/) project's legacy components [Draw2d 3.x](https://www.eclipse.org/gef/draw2d/index.php), [GEF (MVC) 3.x](https://www.eclipse.org/gef/gef_mvc/index.php), and [Zest 1.x](https://www.eclipse.org/gef/zest/index.php), which are still maintained but will not be developed further. The code base of the current GEF production components is located in the [eclipse/gef](https://github.com/eclipse/gef) repository instead.
+This repository contains the code base of the [Eclipse Graphical Editing Framework (GEF)](http://www.eclipse.org/gef/) project's classiic components [Draw2d 3.x](https://www.eclipse.org/gef/draw2d/index.php), [GEF (MVC) 3.x](https://www.eclipse.org/gef/gef_mvc/index.php), and [Zest 1.x](https://www.eclipse.org/gef/zest/index.php).
```

Comment:
there is a small typo in classic. Can I directyl edit that here?

Reply:
Not sure if that work. Try or send another PR

Reply:
It's only possible to give permission to do this to the receiving repositories maintainer, what I read here https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/

Reply:
I googled a bit and it looks like the easiest is if I make a new pull request. But as I would like to update a few more things in the readme it is best if you merge and I work onwards from that.

Reply:
:-)